### PR TITLE
convert to python 3

### PR DESCRIPTION
**Pull request by fawaf**
this update is not backwards-compatible with python2

supercedes #1

### CODE REVIEW

**Comment by csivanich** [comment]
Will test today, but looks like it'll work.

File: dux.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```
Comment (csivanich) [inline dux.py]:
```suggestion
#!/usr/bin/env python3
```
On Ubuntu 18.04 env python is still 2.7, will require explicit setting of 3:
```
 c@mu ~: which python                                                    20:28:45 99% (2.4.0p0)
/usr/bin/python
 c@mu ~: python --version                                                20:28:47 99% (2.4.0p0)
Python 2.7.17
```

**Review by csivanich — CHANGES_REQUESTED** [review]

**Comment by fawaf** [comment]
poketh

**Comment by fawaf** [comment]
poketh @csivanich

**Comment by fawaf** [comment]
poke @csivanich

**Comment by fawaf** [comment]
@csivanich poketh

**Pull request closed without merging**